scripts/buf_lint_check.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+echo "********************************"
+echo "Checking Protobuf file formatting"
+echo "********************************"
+
+../oppia-android-tools/buf-Darwin-x86_64 check lint --input=model/src/main/proto --input-config buf.yaml
```

Comment:
Is it possible to run buf on the whole repository rather than segmenting it to src/main/proto?